packages/xml-views-quick-fix/src/quick-fix-stable-id.ts

```diff
@@ -18,12 +19,16 @@ export type QuickFixStableIdInfo = {
 };
 
 export function computeQuickFixStableIdInfo(
-  xmlDoc: XMLDocument,
+  context: Context,
   errorOffset: OffsetRange[]
 ): QuickFixStableIdInfo[] {
   const biggestIdsByElementNameCollector =
     new BiggestIdsByElementNameCollector();
-  accept(xmlDoc, biggestIdsByElementNameCollector);
+  const files = Object.keys(context.viewFiles);
```

Comment:
See the comment for `validateNonUniqueID`, the same applies here.

Reply:
see above.